section2-week1/exercise 8/Main.cpp

```diff
@@ -19,43 +19,24 @@ int main()
 		yes, no, maybe, perhaps
 	};
 	//inicialize variables
-	answers ans;
-	ans = yes;
+	answers ans = yes;
 
 
-	int v1;
+	int random;
 
 	//selecting the answer
 	srand(time(NULL));
-	v1 = rand() % (1 - 5);
+	random = rand() % (1 - 5);
```

Comment:
I think this was commented before, but why are your using `1 - 5`? That's the same thing that `rand() % -4` so why not use that instead? But even so, `-4` is a bit unnecessarily specific, because you could just as well do `rand() % 4`